twingate/resource_connector.go

```diff
@@ -71,8 +71,6 @@ func resourceConnectorCreate(ctx context.Context, resourceData *schema.ResourceD
 	resourceData.SetId(connector.ID.(string))
 	log.Printf("[INFO] Created conector %s", connector.Name)
 
-	waitForResourceAvailability()
```

Comment:
This change is not related to Groups. Keep this and remove it in the PR that fixes Connectors